.evergreen/install-dependencies.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
I'm starting to worry about the duplication of all of these scripts, does it make sense to just curl them from github during the evergreen run?

Reply:
I'm on the fence about that - also not a big fan of the duplication, but it's only a handful of copies and I think this is the last one, at least for now. If we were to de-duplicate, which repository were you thinking of putting them in?

Reply:
No idea, let's defer to some later date when it actually causes us pain (if it does)